docs/README.md

```diff
@@ -1,6 +1,20 @@
 # Zer0Note
 
-Zer0Note is a *note taking and organisation* app that combines the features of OneNote & Notion with the speed of vim & emacs​.
+```
+ ______ 
+|      |  
+| 0 u 0|    Hello I'm Zer0!
+|      | 
+|______|
+```
```

Comment:
LOL

Reply:
honestly, not sure if i should put it there when we haven't used the mascot anywhere else... and it's not like the mascot is even necessary